tests/v4l2decode.cpp

```diff
@@ -321,9 +323,9 @@ int main(int argc, char** argv)
     fd = YamiV4L2_Open("decoder", 0);
     ASSERT(fd!=-1);
 
+#if __ENABLE_V4L2_GLX__
     x11Display = XOpenDisplay(NULL);
```

Comment:
eglInit need this to call eglGetDisplay. it will break VIDEO_DATA_MEMORY_TYPE_DMA_BUF & VIDEO_DATA_MEMORY_TYPE_DRM_NAME in no glx mode.